content/machine-controller/main/cloud-providers/vsphere/template-vm/qcow2/_index.en.md

```diff
@@ -27,7 +27,7 @@ but VM Templates are not supported yet by machine-controller.
 
     ```bash
     # The URL below is just an example
-    image_url="https://cloud.centos.org/centos/7/images/CentOS-7-x86_64-GenericCloud.qcow2"
+    image_url="https://cloud.ubuntu.org/ubuntu/7/images/Ubuntu-x86_64-GenericCloud.qcow2"
```

Comment:
let's please use a correct URL for Ubuntu images here, even though this is just an example.